rllib/connectors/connector.py

```diff
@@ -94,11 +94,14 @@ class Connector(abc.ABC):
     """
 
     def __init__(self, ctx: ConnectorContext):
-        # This gets flipped to False for inference.
+        # Default is training mode.
+        self.in_training()
```

Comment:
nit: PyCharm probably complains here and I think it's a good convention to create variables inside __init__!

Reply:
this is a good call. thanks.